extra/1-radio-stations.js

```diff
@@ -29,7 +33,13 @@ function getAllFrequencies() {
  */
 // `getStations` goes here
 function getStations () {
-  
+ let radioStFreq =[]; 
+let activeFrequency  = getAllFrequencies();
+for (i=0; i<=activeFrequency.length; i++);
```

Comment:
You also need the keyword var in front of the i=0 on line 38. At the end of the same line you don't need a semicolon. Instead of semicolon you will have to open curly brackets so the computer knows where the statement  of the for loop starts and ends. It should end on line 42 so don't forget to close curly brackets before the return.